Gemfile

```diff
@@ -10,6 +10,6 @@ end
 group :release, optional: true do
   gem 'faraday-retry', '~> 2.1', require: false
   # fix from smortex to properly process commits that exist in multiple branches
-  # gem 'github_changelog_generator', github: 'smortex/github-changelog-generator', branch: 'avoid-processing-a-single-commit-multiple-time', require: false
-  gem 'github_changelog_generator', '~> 1.16.4', require: false
+  gem 'github_changelog_generator', github: 'smortex/github-changelog-generator', branch: 'avoid-processing-a-single-commit-multiple-time', require: false
```

Comment:
Shall we maybe
1. Move "smortex/github-changelog-generator" to voxpupuli
2. Release it
3. Use the released version here

Reply:
hoping that it will be fixed upstream, I switched back to the original release.